### PR TITLE
Add profile required modules in the .info file as dependencies

### PR DESCRIPTION
**Pull request by klonos**
https://github.com/PackWeb/pw_profile/issues/5

### CODE REVIEW

File: pw_profile.info [hunk 1]
```diff
@@ -3,3 +3,29 @@ description = Configure Backdrop for a new PackWeb site.
...
+dependencies[] = imgareaselect
+dependencies[] = invisimail
+dependencies[] = libraries
+dependencies[] = maintenance_page
+dependencies[] = manualcrop
+dependencies[] = menu_admin_per_menu
+dependencies[] = menu_trail_by_path
+dependencies[] = on_the_web
+dependencies[] = pathologic
+;pw_module
```
Comment (klonos) [inline pw_profile.info]: Not sure what this is.

Comment (BWPanda) [reply pw_profile.info]: https://github.com/PackWeb/pw_module

**Comment by BWPanda** [comment]
@klonos I've created a separate branch for this to test: https://github.com/PackWeb/pw_profile/tree/info-dependencies
I've also added a few extra modules there you may have missed, and disabled the module-enabling function elsewhere in the profile.
Let me know how that works for you.

**Comment by BWPanda** [comment]
As per https://github.com/PackWeb/pw_profile/issues/7 I have gone a different route by including a script users can run that downloads all modules, etc. needed to use this profile. They still aren't included as dependencies (since that'd mean duplication), but I don't see that as a big need ATM. The seperate branch mentioned above will now be deleted to focus on improving the current system.

**Pull request closed without merging**